schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1014 @@
+      "items": {
+        "type": "string",
+        "examples": ["6000456", "8004567"]
+      },
+      "description": "A list of related case references known to the appellant and the LPA"
+    },
+    "neighbouringSiteAddresses": {
+      "type": ["array", "null"],
+      "items": {
+        "type": "object",
+        "additionalProperties": true,
+        "properties": {
+          "neighbouringSiteAddressLine1": {
+            "type": "string",
+            "examples": ["96 The Avenue"],
+            "description": "First line of address of the site"
+          },
+          "neighbouringSiteAddressLine2": {
+            "type": ["string", "null"],
+            "description": "Second line of address of the site"
+          },
+          "neighbouringSiteAddressTown": {
+            "type": "string",
+            "examples": ["Maidstone"],
+            "description": "Town / City of the site address"
+          },
+          "neighbouringSiteAddressCounty": {
+            "type": ["string", "null"],
+            "examples": ["Kent"],
+            "description": "County of the site address"
+          },
+          "neighbouringSiteAddressPostcode": {
+            "type": "string",
+            "examples": ["MD21 5XY"],
+            "description": "Postal code of the site address"
+          },
+          "neighbouringSiteAccessDetails": {
+            "type": ["string", "null"],
+            "examples": ["Ongoing road works at this address"],
+            "description": "Provided information on site accessibility on this address"
+          },
+          "neighbouringSiteSafetyDetails": {
+            "type": ["string", "null"],
+            "examples": ["Ongoing road works at this address"],
+            "description": "Provided information on site health and safety on this address"
+          }
+        },
+        "required": [
+          "neighbouringSiteAddressLine1",
+          "neighbouringSiteAddressLine2",
+          "neighbouringSiteAddressTown",
+          "neighbouringSiteAddressCounty",
+          "neighbouringSiteAddressPostcode",
+          "neighbouringSiteAccessDetails",
+          "neighbouringSiteSafetyDetails"
+        ]
+      },
+      "description": "A list of neighbouring site addresses"
+    },
+    "affectedListedBuildingNumbers": {
+      "type": ["array", "null"],
+      "items": {
+        "type": "string",
+        "examples": ["10023", "17824"]
+      },
+      "description": "A list of affected listed building IDs from Historic England"
+    },
+    "appellantCostsAppliedFor": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the appellant has applied for costs"
+    },
+    "lpaCostsAppliedFor": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the appellant has applied for costs"
+    },
+    "agriculturalHolding": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the site is an agricultural holding",
+      "$comment": "S78"
```

Comment:
I see the thinking here, but it is actually more helpful to have the inverse? Since all fields in this schema are S78?

`"$comment": "also on HAS cases"` or something for the common fields